app/javascript/src/components/GameLog.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="border border-black rounded p-2 m-2">
+    
+    <div>
+      <GameLogEntry
+        v-for="(entry, index) in processedLog()"
+          :events="entry"
+          :key="index"
+      ></GameLogEntry>
+    </div>
+  </div>
+</template>
+
+<script>
+import GameLogEntry from "./GameLogEntry.vue";
+export default {
+  name: "GameLog",
+  props: { log: Array },
+  components: { GameLogEntry },
+  methods: {
+    processedLog() {
+      let rawlog = this.log || [];
+      let entries = [];
+      rawlog.forEach((action) => {
+        if (action.type === "initialize") {
+          entries.push({event: action});
+        } else if (action.type === "rondel") {
+          entries.push({event: [action]});
```

Comment:
I wonder if this can be tightened up like this:
```javascript
if (action.type === "initialize" || action.type === "rondel") {
  entries.push({event: [action]});
}
```
This would be a substantive change in that the `action` for "initialize" would be a single element in an array rather than directly the element itself. But I feel like that will keep the pattern a little tighter. Would my suggested change break things?

Reply:
Yes, it would need some other changes for this to work but it's worth it.